tox.ini

```diff
@@ -114,6 +115,30 @@ commands =
     rm -rf {envdir}/.pytest_cache
     make regen
 
+[testenv:plugins]
+changedir = testing/plugins_integration
+deps =
+    django
+    pytest-asyncio
+    pytest-cov
+    pytest-django
+    pytest-html
+    pytest-mock
+    pytest-sugar
+    pytest-trio
+    anyio[curio,trio]
+setenv =
+    PYTHONPATH=.
+commands =
+    pytest pytest_asyncio_integration.py
+    pytest --cov=. simple_integration.py
+    pytest --ds=django_settings simple_integration.py
+    pytest --html=simple.html simple_integration.py
+    pytest pytest_mock_integration.py
+    pytest simple_integration.py --force-sugar
+    pytest pytest_trio_integration.py
+    pytest pytest_anyio_integration.py
```

Comment:
I think we should test all the async frameworks simultaneously - they have all agreed to use marks to allow them to all interoperate in a single test run, so we should test that here

Reply:
At least as long as you don't use async fixtures. `pytest-asyncio` is known to appropriate those regardless of marks (see https://github.com/pytest-dev/pytest-asyncio/issues/124 for the issue).

Reply:
> At least as long as you don't use async fixtures. 

That's a very good point, we currently don't test any async fixtures here.